pkg/publisher/fanout/publisher.go

```diff
@@ -136,15 +136,15 @@ func (f *fanoutPublisher) ValidateJob(ctx context.Context, j models.Job) error {
 // PublishResult implements publisher.Publisher
 func (f *fanoutPublisher) PublishResult(
 	ctx context.Context,
-	executionID string,
+	execution models.Execution,
 	job models.Job,
```

Comment:
we can stop passing the job since the execution also holds job instance. Also I've been passing pointers of large objects instead of a copy. I understand that also has its own set of drawbacks and potentially impacting data integrity, but it is not that harmful when we know that the source (boltdb) is giving us a copy and not the source object

Reply:
Resolved in d330f8f0